src/renderers/dom/shared/syntheticEvents/SyntheticUIEvent.js

```diff
@@ -19,23 +19,7 @@ var getEventTarget = require('getEventTarget');
  */
 var UIEventInterface = {
   view: function(event) {
-    if (event.view) {
-      return event.view;
-    }
-
-    var target = getEventTarget(event);
-    if (target.window === target) {
-      // target is a window object
-      return target;
-    }
-
-    var doc = target.ownerDocument;
-    // TODO: Figure out why `ownerDocument` is sometimes undefined in IE8.
-    if (doc) {
-      return doc.defaultView || doc.parentWindow;
-    } else {
-      return window;
-    }
+    return event.view;
   },
   detail: function(event) {
     return event.detail || 0;
```

Comment:
I believe, unless I'm mistaken, this can be:

```
var UIEventInterface = {
  view: null,
  detail: null
}
```

Reply:
Maybe the var can be completely removed then? and not augmenting the class?